### ISSUE TITLE
On BigSur, Windows10-VM freezes at Start Boot Option

### ISSUE THREAD

**Issue by Maxiggapp** [utmapp/UTM#2497]
**Describe the issue**
After installing Windows10-ISO i start the VM. But it does not booting. VM freezes at Start Boot Option.

**Configuration**
* UTM Version: Version 2.0.27 (27)
* OS Version: BigSur 11.3.1
* Intel or Apple Silicon? Intel i3 Quad-Core

**Upload VM**
`config.plist`

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>BuildMachineOSBuild</key>
	<string>19H524</string>
	<key>CFBundleDevelopmentRegion</key>
	<string>en</string>
	<key>CFBundleDocumentTypes</key>
	<array>
		<dict>
			<key>CFBundleTypeName</key>
			<string>UTM virtual machine</string>
			<key>CFBundleTypeRole</key>
			<string>Editor</string>
			<key>LSHandlerRank</key>
			<string>Owner</string>
			<key>LSItemContentTypes</key>
			<array>
				<string>com.utmapp.utm</string>
			</array>
			<key>LSTypeIsPackageLSTypeIsPackage</key>
			<true/>
		</dict>
	</array>
	<key>CFBundleExecutable</key>
	<string>UTM</string>
	<key>CFBundleIconFile</key>
	<string>AppIcon-macOS</string>
	<key>CFBundleIconName</key>
	<string>AppIcon-macOS</string>
	<key>CFBundleIdentifier</key>
	<string>com.utmapp.UTM</string>
	<key>CFBundleInfoDictionaryVersion</key>
	<string>6.0</string>
	<key>CFBundleName</key>
	<string>UTM</string>
	<key>CFBundlePackageType</key>
	<string>APPL</string>
	<key>CFBundleShortVersionString</key>
	<string>2.0.27</string>
	<key>CFBundleSupportedPlatforms</key>
	<array>
		<string>MacOSX</string>
	</array>
	<key>CFBundleVersion</key>
	<string>27</string>
	<key>DTCompiler</key>
	<string>com.apple.compilers.llvm.clang.1_0</string>
	<key>DTPlatformBuild</key>
	<string>12B45b</string>
	<key>DTPlatformName</key>
	<string>macosx</string>
	<key>DTPlatformVersion</key>
	<string>11.0</string>
	<key>DTSDKBuild</key>
	<string>20A2408</string>
	<key>DTSDKName</key>
	<string>macosx11.0</string>
	<key>DTXcode</key>
	<string>1220</string>
	<key>DTXcodeBuild</key>
	<string>12B45b</string>
	<key>ITSAppUsesNonExemptEncryption</key>
	<false/>
	<key>LSApplicationCategoryType</key>
	<string>public.app-category.business</string>
	<key>LSMinimumSystemVersion</key>
	<string>11.0</string>
	<key>NSAccentColorName</key>
	<string>AccentColor</string>
	<key>UTExportedTypeDeclarations</key>
	<array>
		<dict>
			<key>UTTypeConformsTo</key>
			<array>
				<string>com.apple.package</string>
			</array>
			<key>UTTypeDescription</key>
			<string>UTM virtual machine</string>
			<key>UTTypeIcons</key>
			<dict>
				<key>UTTypeIconText</key>
				<string>UTM</string>
			</dict>
			<key>UTTypeIdentifier</key>
			<string>com.utmapp.utm</string>
			<key>UTTypeTagSpecification</key>
			<dict>
				<key>public.filename-extension</key>
				<array>
					<string>utm</string>
				</array>
			</dict>
		</dict>
	</array>
</dict>
</plist>

**Comment by conath** [comment]
You have included the info.plist in your issue instead of the Config.plist.

If it is easier for you, please use the Export QEMU Command button in VM Config → QEMU. This usually includes all the necessary info. Thanks.

**Comment by Maxiggapp** [comment]
Thank you, here is the Export of QEMU:

qemu-system-aarch64 -L /Applications/UTM.app/Contents/Resources/qemu -S -qmp tcp:127.0.0.1:4444,server,nowait -vga none -spice port=5930,addr=127.0.0.1,disable-ticketing,image-compression=off,playback-compression=off,streaming-video=off -device virtio-ramfb -cpu cortex-a72 -smp cpus=1,sockets=1,cores=1,threads=1 -machine virt,highmem=on -accel tcg,tb-size=4096 -bios /Applications/UTM.app/Contents/Resources/qemu/edk2-aarch64-code.fd -m 16384 -device intel-hda -device hda-duplex -name "VM-1 Win10" -device qemu-xhci -device usb-tablet -device usb-mouse -device usb-kbd -device nvme,drive=drive0,serial=drive0,bootindex=0 -drive "if=none,media=disk,id=drive0,file=/Users/guidoglaser/Library/Containers/com.utmapp.UTM/Data/Documents/VM-1 Win10.utm/Images/Windows10_InsiderPreview_Client_ARM64_en-us_21354.VHDX,cache=writethrough" -device usb-storage,drive=drive1,removable=true,bootindex=1 -drive if=none,media=cdrom,id=drive1 -device virtio-net-pci,mac=42:3C:73:31:28:55,netdev=net0 -netdev user,id=net0 -device virtio-serial -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -uuid 135D4C5E-779F-4D29-AAAA-FDF40A7C7270 -rtc base=localtime

**Comment by conath** [comment]
Thanks for this, it contains most of the information I need. Firstly, it looks like you are trying to start the ARM version of Windows 10 on your Intel Mac mini. Did you mean to do this or would you be happy with the x86_64 (Intel) version of Windows 10? 

By using the ARM version, what happens technically is that UTM needs to do lots of work behind the scenes to emulate the ARM CPU on your Intel i3, which means the outcome is very slow performance. I suspect that the VM would eventually boot up if you waited for a long time. 

To download Windows 10 Intel version, go to [Microsoft.com](https://www.microsoft.com/en-us/software-download/windows10ISO).

**Comment by gayanper** [comment]
> The configuration is not right. Please try these VM templates, click here for [Windows 10](https://chrisp.cafe/UTM/Windows_10_x64.utm.zip) and [here for Ubuntu](https://chrisp.cafe/UTM/Ubuntu_Desktop_20.04.utm.zip).

@conath is this windows template supposed to work on MacOS as well ? when i tried it stops at the prompt without booting the attached installation win10 iso. The iso i tried is x86 and i'm trying on a macos x86 version.

And the source for these templates are (the host) is it something you maintain is it ? just wondering the the safety :).

**Comment by conath** [comment]
> @conath is this windows template supposed to work on MacOS as well ? when i tried it stops at the prompt without booting the attached installation win10 iso. The iso i tried is x86 and i'm trying on a macos x86 version.

Yes, it's supposed to work on Intel Mac host. On my Mac, I can download that template, double click and then choose the ISO for the DVD drive. When I start the VM after that, it shows a couple of messages and then Windows starts with "Press any key to boot from CD/DVD". You need to press a key (e. g. space) to start booting the installer.
If yours gets stuck at EFI shell, you can try to press esc at the UTM logo screen and then choose Boot Manager, then select `UEFI QEMU DVD-ROM` to boot from the ISO. (you will still be prompted to press any key to boot)

> And the source for these templates are (the host) is it something you maintain is it ? just wondering the the safety :).

Chrisp.cafe is maintained by me.

**Comment by gayanper** [comment]
@conath Thanks, i tried your template, but i didn't workout. I also downloaded the virtio storage drivers and load it to installer, but the installer always crash with the error which says something about not equal. I think i saw the same message in one of the issues here.

Then i tried installing qemu through brew and use the following commands in https://superuser.com/questions/905815/how-to-setup-windows-7-in-qemu-on-mac and it worked for windows 10 as well. I tried additionally passing -accel hvf and the installation went just fine.

Is there any reason why qemu under UTM didn't worked ?
